sources/web/datalab/templates/nb.html

```diff
@@ -199,37 +180,16 @@
         </div>
         <div id="sidebarArea" style="display:none">
           <div id="sidebarContent">
-            <div class="btn-group">
-              <button id="toggleSidebarButton" type="button" class="toolbar-btn" title="Toggle the size of the sidebar">
-                <i class="material-icons">chevron_left</i>
-              </button>
+            <div>
+              <div class="btn-group pull-right">
```

Comment:
Done but I'm not sure if I have `overflow: auto` in the right place. The 'X' button looks better now. It's possible that the issue is fixed by using `btn btn-default` instead of `toolbar-btn`.